English_transcripts/school/Epic Quest/sharedAssets/C1_E_EQ_WProbs_demo1.txt

```diff
@@ -0,0 +1 @@
+I have some problems that i will ask you to solve for me. Here are some things to help you. You can use them if you need them but you dont have to use them. Listen very carefully to each problem. Tap this button and i will repeat problem for you. Okay lets get started. 
```

Comment:
Tap this button and i will repeat the problem for you.

Reply:
Updated.